### DISCUSSION TITLE
xtreme codes for Fermata Auto

### DISCUSSION THREAD

**Discussion by leefender** [AndreyPavlenko/Fermata#211] [Q&A]
Hi Andrey, is there a plan to add the ability to ender xtreme codes logins for iptv in the future. Most providers no longer give the M3u playlist for paid iptv services. 

I'd really like to be able to login to my iptv and web iptv smarters is practically unusable on fermata

Thanks in advance

Lee

**Comment by AndreyPavlenko** [comment]
Unfortunately, I don't have time to implement this feature right now, sorry.
But this is an open source project. If someone could contribute to this feature, the entire community would be greatly appreciated.

**Reply by leefender** [reply]
Thanks for the quick response, regardless this is a phenomenal app, hopefully someone has the time. In future to add this feature.

Thanks again

**Reply by DrUnKeNTiGeR** [reply]
I second this

**Reply by leefender** [reply]
> I second this

I've since discovered Moul TV will give you the M3u so I have bought a year sub to them solely for the car and i have it working
